src/selectors/prometheus/selectors.js

```diff
@@ -28,3 +28,8 @@ export const getLastUtilizationStat = response => {
   const history = getUtilizationVectorStats(response);
   return history ? history[history.length - 1] : null;
 };
+
+export const getDataResiliencyData = response => {
```

Comment:
this function is the same as `getCapacityStats`. Can you please rename it and use it instead?

Reply:
done